src/PerimeterxHttpClient.php

```diff
@@ -24,6 +24,7 @@ class PerimeterxHttpClient
     public function __construct($config)
     {
         $this->client = new Client(['base_uri' => $config['perimeterx_server_host']]);
+        $this->captcha_client = new Client(['base_uri' => $config['captcha_script_host']]);
```

Comment:
i would consider using the client as this is php and it doesn't save stickiness, not necessarily creating more than 1 client
http://docs.guzzlephp.org/en/stable/quickstart.html